src/Traits/PropertyTypeTrait.php

```diff
@@ -39,32 +39,35 @@ trait PropertyTypeTrait
      * Perform an equality check or an assignment with the given expression.
      *
      * @param PropertyType $right
+     * @param bool $insertParentheses
      * @return Equality
      */
-    public function equals(PropertyType $right): Equality
+    public function equals(PropertyType $right, bool $insertParentheses = true): Equality
```

Comment:
The `$insertParentheses` parameter should also be added to the relevant interface (`PropertyType`).

Reply:
fixed in 03246a1